File: src/com/qx/day07/day07.java
package com.qx.day07;

public class day07 {

    public static void main(String[] args) {
        printText();
        System.out.println(printTextForTitle("这个是带返回值的方法"));
    }

    protected static void printText() {
        System.out.println("这个是没有返回值的方法");
    }

    protected static String printTextForTitle(String title){

        return title;
    }
}

File: src/com/qx/day07/day07.kt
package com.qx.day07


fun main() {
    printText()
    println(printTextForTitle("这个是带返回值的方法"))
}

fun printText() {
    println("这个是没有返回值的方法")
}

fun printTextForTitle(title: String?): String? {
    return title
}